content/codepipeline/codepipeline.md

```diff
@@ -21,7 +21,7 @@ Click the **Launch** button to create the CloudFormation stack in the AWS Manage
 
 | Launch template |  |  |
 | ------ |:------:|:--------:|
-| CodePipeline & EKS |  {{% cf-launch "ci-cd-codepipeline.cfn.yml" "eksws-codepipeline" %}} | {{% cf-download "ci-cd-codepipeline.cfn.yml" %}}  |
+| CodePipeline & EKS |  {{% cf-launch "eksws-codepipeline" "ci-cd-codepipeline.cfn.yml" %}} | {{% cf-download "ci-cd-codepipeline.cfn.yml" %}}  |
```

Comment:
not sure if this also needs to be reverted since reverting the commit in cf-launch?